content/master/concepts/claim-server-side-apply.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Claim Server-Side Apply
+state: alpha
+alphaVersion: "1.15"
+weight: 61
+---
+
+When you create a claim, Crossplane creates a corresponding composite resource
+(XR). Crossplane keeps the claim in sync with the composite resource. When you
+change the claim, Crossplane reflects those changes on the composite resource.
+
+Read the [claims documentation]({{<ref "./claims">}}) to learn more about claims
+and how they relate to composite resources.
+
+Crossplane can use server-side apply to keep the claim in sync with the
+composite resource.
+
+{{<hint "tip">}}
+Use the `--enable-claim-ssa` feature flag to enable using server-side apply.
```

Comment:
by looking at the crossplane source code, it looks that the flag name is actually `--enable-ssa-claims`, right @negz ?

Reply:
Haha yes it is. Good catch. 🙃